help/deploy/releases/release-history.md

```diff
@@ -22,6 +22,132 @@ See also:
 - [Docker-Compose](../docker-compose/docker-compose.md)
 
 
+## 2024-09-10 Datagrok 1.21.1 release 
+
+The Datagrok 1.21.1 release enhances platform stability and usability, streamlining workflows for a more intuitive user experience. 
+
+Database migrations are unrevertable in this version. You can't rollback to older DF version after first 1.21.0 run. If you still need to rollback - contact DG support for the assistance.
```

Comment:
I didn't include Jira ticket numbers because they are internal, and users can't access our Jira. The Jira ticket number is not useful for the customer, so removing them made the notes more concise.

However, GitHub tickets are open-access, and the links to the issues are preserved.

 If you think we should include Jira tickets as well, I'll add them back.

Reply:
Changed to: Database migrations are irreversible in this version. You cannot roll back to an older version of Datagrok after the first 1.21.0 run. If a rollback is necessary, please contact Datagrok support for assistance.

Reply:
Ok